LRUCache11.hpp

```diff
@@ -204,6 +204,7 @@ struct List {
 			head = n;
 		} else if (head == tail) {
 			head->next = n;
+			n->prev = head;
```

Comment:
To be honest, you can probably just drop this whole `if` block - when `head == tail`, the behaviour is the same as simply appending to `tail` anyway.